lib/backend/model/workloadendpoint.go

```diff
@@ -71,7 +71,9 @@ func (key WorkloadEndpointKey) defaultDeleteParentPaths() ([]string, error) {
 	workload := fmt.Sprintf("/calico/v1/host/%s/workload/%s/%s",
 		key.Hostname, escapeName(key.OrchestratorID), escapeName(key.WorkloadID))
 	endpoints := workload + "/endpoint"
-	return []string{endpoints, workload}, nil
+	orchestrator := fmt.Sprintf("/calico/v1/host/%s/workload/%s",
+		key.Hostname, escapeName(key.OrchestratorID))
+	return []string{endpoints, workload, orchestrator}, nil
```

Comment:
I may have misunderstood the issue, but this was quick to implement so I figured why not, now you've given me a why not :) will remove